contracts/proposal/cwd-proposal-multiple/src/contract.rs

```diff
@@ -777,19 +862,77 @@ pub fn reply(deps: DepsMut, env: Env, msg: Reply) -> Result<Response, ContractEr
             Ok(Response::new().add_attribute("proposal execution failed", proposal_id.to_string()))
         }
         TaggedReplyId::FailedProposalHook(idx) => {
-            let addr = PROPOSAL_HOOKS.remove_hook_by_index(deps.storage, idx)?;
-            Ok(Response::new().add_attribute("removed proposal hook", format!("{addr}:{idx}")))
+            // When firing off proposal hooks indexes `[0,
+            // hook_count)` are used to represent hooks that have been
+            // fired via the regular PROPOSAL_HOOKS method. Index
+            // `hook_count` is used to represent the pre-propose
+            // module.
```

Comment:
looks like this is still using the old hooks logic :)

Reply:
uh oh, sorry thought I updated that